### DISCUSSION TITLE
2024/12/17/%E6%80%BB%E7%BB%9F%E8%B5%A6%E5%85%8D%E8%87%AA%E5%B7%B1%E5%84%BF%E5%AD%90

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#1] [Announcements]
# 2024/12/17/%E6%80%BB%E7%BB%9F%E8%B5%A6%E5%85%8D%E8%87%AA%E5%B7%B1%E5%84%BF%E5%AD%90

前两天在知乎上看到一个有趣的问题：“拜登赦免了自己的儿子，这么明显的制度漏洞，为什么没有受到其他权力的制约？三权分立失效了吗？”

https://blog.qizhen.xyz/2024/12/17/%E6%80%BB%E7%BB%9F%E8%B5%A6%E5%85%8D%E8%87%AA%E5%B7%B1%E5%84%BF%E5%AD%90

**Comment by ruanqizhen** [comment]
测试留言
